projects/mercury/src/metadata/views/MetadataView.js

```diff
@@ -141,59 +141,69 @@ export const MetadataView = (props: MetadataViewProperties) => {
         return view.columns;
     };
 
+    const renderSingleFacet = (facet) => {
+        const facetOptions = ofRangeValueType(facet.type) ? [facet.min, facet.max] : facet.values;
+        const activeFilter = [...filterCandidates, ...filters].find(filter => filter.field === facet.name);
+        let activeFilterValues = [];
+        if (activeFilter) {
+            activeFilterValues = ofRangeValueType(facet.type) ? [activeFilter.min, activeFilter.max] : activeFilter.values;
+        }
+        return facetOptions && facetOptions.length > 0 && (
+            <Grid key={facet.name} item>
+                <Facet
+                    type={facet.type}
+                    title={facet.title}
+                    options={facetOptions}
+                    onChange={(values) => updateFilterCandidates(facet, values)}
+                    extraClasses={classes.facet}
+                    activeFilterValues={activeFilterValues}
+                    clearFilter={() => handleClearFilter(facet.name)}
+                />
+            </Grid>
+        );
+    };
+
+    const renderFacetConfirmButtons = (
+        <Grid
+            container
+            spacing={1}
+            className={`${classes.confirmFiltersButtonBlock} ${filterCandidates.length > 0 && classes.confirmFiltersButtonBlockActive}`}
+        >
+            <Grid item xs={4}>
+                <Button
+                    onClick={clearFilterCandidates}
+                    variant="contained"
+                    color="default"
+                    className={classes.confirmFiltersButton}
+                    disabled={filterCandidates.length === 0}
+                >
+                    Cancel
+                </Button>
+            </Grid>
+            <Grid item xs={8}>
+                <Button
+                    onClick={applyFilters}
+                    variant="contained"
+                    color="secondary"
+                    className={classes.confirmFiltersButton}
+                    disabled={filterCandidates.length === 0}
+                >
+                    Apply filters
+                </Button>
+            </Grid>
+        </Grid>
+    );
+
     const facetsEx = collectionsFacet ? [...facets, collectionsFacet] : facets;
 
-    const renderFacets = () => (
+    const renderFacets = (view) => (
         <Grid container item direction="column" justifyContent="flex-start" spacing={1}>
```

Comment:
```suggestion
        <Grid key={view.name} container item direction="column" justifyContent="flex-start" spacing={1}>
```
Since these are created by iterating over a list, React expects these components to have a unique key (see the console warnings).